tools/cuvslam_app/README.md

```
@@ -60,6 +60,17 @@ Run tracking on a video file with one loop:
     --enable_observations_export=true
 ```
 
+### PNG to TGA Cache
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)